tests/src/main/java/com/hazelcast/stabilizer/tests/map/helpers/MapStoreWithCounterPerKey.java

```diff
@@ -0,0 +1,44 @@
+package com.hazelcast.stabilizer.tests.map.helpers;
+
+import com.hazelcast.core.MapStore;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Created by dirk on 16/01/15.
+ */
+public class MapStoreWithCounterPerKey extends MapStoreWithCounter {
+
+    public Map<Object, AtomicInteger> storeCount = new ConcurrentHashMap<Object, AtomicInteger>();
+
+    @Override
+    public void store(Object o, Object o2) {
```

Comment:
To improve readability, please rename `o` to `key` and `o2` to `value` like in the interface.